compiler/rustc_metadata/src/rmeta/decoder.rs

```diff
@@ -76,9 +77,12 @@ impl MetadataBlob {
 /// own crate numbers.
 pub(crate) type CrateNumMap = IndexVec<CrateNum, CrateNum>;
 
-/// Target modifiers - abi or exploit mitigations flags
+/// Target modifiers - abi or exploit mitigations flags that cause unsoundness when mixed
 pub(crate) type TargetModifiers = Vec<TargetModifier>;
 
+/// Enforced Mitigations
+pub(crate) type EnforcedMitigations = Vec<EnforcedMitigation>;
```

Comment:
Would it be easier to have a list of allowed partial mitigations instead (with all mitigations being enforced by default)?

Reply:
> Would it be easier to have a list of allowed partial mitigations instead (with all mitigations being enforced by default)?

An "enforced mitigation" is (to me) the name of a mitigation that is enforced according to RFC 3855. Maybe we should call it "EnforcableMitigation"?

In metadata, you need to have the list of enforced mitigations that the dependency crate has enabled.